sdk/core/core-auth/src/tokenCredential.ts

```diff
@@ -25,6 +25,10 @@ export interface TokenCredential {
  * Defines options for TokenCredential.getToken.
  */
 export interface GetTokenOptions {
+  /**
+   * Additional claims to be included in the token. See {@link https://openid.net/specs/openid-connect-core-1_0-final.html#ClaimsParameter} for more information on format and content.
+   */
+  claims?: string;
```

Comment:
Since CAE has been pushed out of the current milestone, am guessing this is not needed for the Feb release? @sadasant Let's make sure `@azure/core-auth` is released before merging this PR

Reply:
I am not sure if this needs to be released or not. It seemed like it needed to be released. @schaabs please confirm.

Reply:
It needs to be released before we can release any CAE support in core-http or identity. So it doesn't have to be released now, but if not we'll need to include this with our out of band releases for CAE support.  All that said, I don't see any harm in releasing it now.

Reply:
CAE support would be in beta, and we are yet to discuss whether core-auth would ship a beta or a patch/minor update to support the CAE features. So, my recommendation would be to release the core-auth from the master branch so that Event Grid is unblocked, and then have an out of band release for core-auth